doc/languages-frameworks/qt.xml

```diff
@@ -113,6 +113,16 @@ mkDerivation {
 </programlisting>
  </para>
 
+ <note>
+  <para>
+    <literal>wrapQtAppsHook</literal> ignores files that are not ELF
```

Comment:
```suggestion
    <literal>wrapQtAppsHook</literal> ignores files without ELF
```

Reply:
What about non-ELF binaries? Or should we we not `ELF` at all since it's possible people don't know what that means?

Reply:
Let's say "non-ELF executables (for example, scripts)": that presents all the technical information an expert needs, but it also presents the information that a non-expert most likely needs (that their script is intentionally not wrapped).

Reply:
I switched it to `non-ELF executables` but I don't think we should just make it one sentence.
Usually really terse sentences like that are glazed over by non-experts.

Reply:
it's still accurate terms, if they are unsure, they can look it up with good SEO (high accuracy about what ELF binaries are).

For non-experts, i think the `(for example, scripts)` is relatable enough to give them all the context they need